packages/bot-web-ui/src/pages/tutorials/__tests__/tutorial-tab-mobile.spec.tsx

```diff
@@ -138,4 +138,23 @@ describe('<TutorialsTabDesktop />', () => {
 
         expect(inputElement.value).toBe(checkinputValue);
     });
+
+    it('should make the search visible on click of onHandleChangeMobile()', () => {
+        const search_icon = screen.getByTestId('search-icon');
+
+        userEvent.click(search_icon);
+        const search = screen.getByTestId('id-search-visible');
+
+        expect(search).toHaveClass('tutorials-mobile__select--show-search');
+    });
+
+    it('should change active_tutorial on click of onChangeHandle()', () => {
+        const select = screen.getByTestId('id-tutorials-select');
+        const prev_active_tutorials = mock_DBot_store.dashboard.active_tab_tutorials;
+
+        fireEvent.change(select, { target: { value: 'Quick strategy guides' } });
+        const last_active_tutorials = mock_DBot_store.dashboard.active_tab_tutorials;
```

Comment:
@maryia-matskevich-deriv any chance we could stick to userEvent here?

Reply:
@shafin-deriv using `userEvent.type() `(for event change) doesn't show a passed case of coverage `Branches`, as if the event hadn't happened. If you know how to make the necessary check with `userEvent.type() `- I will be only glad. For me it doesn't work as expected🤨 You may have noticed I always use `userEvent` methods as more modern thing